tests/Unit/Visualization/Python/Test_InterpolateVolumeData.py

```diff
@@ -19,18 +22,19 @@ def setUp(self):
         with two tensor components each
         """
 
-        self.path = unit_test_build_path()
-        self.file_name = os.path.join(self.path, "interpolation_1.h5")
+        self.test_dir = os.path.join(unit_test_build_path(), "Visualization",
```

Comment:
I faintly remember there was some reason we did not use `tempfile` here, was it python2 support or portability? Can we use it now?

Reply:
There's also `click.isolated_filesystem()`. I haven't looked into this yet, but could be nice